recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb

```diff
@@ -9,24 +9,22 @@ LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384
 
 SRC_URI = "\
     file://domu-m3ulcb.cfg \
-    file://domu-h3ulcb.cfg \
-    file://domu-salvator-x-m3.cfg \
-    file://domu-salvator-x-h3.cfg \
+    file://domu-generic-h3.cfg \
     file://domu-salvator-xs-m3n.cfg \
     file://domu-salvator-x-h3-4x2g.cfg \
     file://guest_domu \
 "
 
 S = "${WORKDIR}"
 
-DOMU_CONFIG_salvator-x-m3-xt = "domu-salvator-x-m3.cfg"
-DOMU_CONFIG_salvator-x-h3-xt = "domu-salvator-x-h3.cfg"
+DOMU_CONFIG_salvator-x-m3-xt = "domu-generic-h3.cfg"
```

Comment:
Could we choose some name without "h3" here? xxx-m3 = yyy-h3 sounds a bit confusing.